src/hooks/useFavorites.js

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from 'react-query';
+import api from '../config'
+
+
+export default function useFavorites(booksItemCount, pageNumber, sortBy) {
+
+
+  const getFavorites = api.get(`/favorites`,{booksItemCount, pageNumber, sortBy}) 
+
+    return useQuery(['favorites', pageNumber], () => getFavorites);
+}
```

Comment:
More formatting issues here, return useQuery is indented too far and it looks like the file lacks a "newline" at the end of the file.